projects/aca-shared/rules/src/app.rules.ts

```diff
@@ -448,6 +448,18 @@ export const isTrashcanItemSelected = (context: RuleContext): boolean => [naviga
  */
 export const canViewFile = (context: RuleContext): boolean => [hasFileSelected(context), navigation.isNotTrashcan(context)].every(Boolean);
 
+/**
+ * Checks if user can **Close** opened file.
+ * JSON ref: `canClosePreview`
+ *
+ * @param context Rule execution context
+ */
+export const canClosePreview = (context: AcaRuleContext): boolean => {
```

Comment:
Need to add this 'canClosePreview' in the documentation?
alfresco-content-app/docs/extending/rules.md

Reply:
Done